Indexer.php

```diff
@@ -114,26 +114,32 @@ public function filterIndexedAndNotExpired($uris, array $metadata = [])
         $this->setCoreNameFromMetadata($metadata);
 
         $currentDate = new DateTime();
-
         $docIds = $this->getUniqueHashIds($uris);
-        $queryPhrase = sprintf(
-            'id:(%s) AND revisit_expiration:[%s TO *]',
-            implode(' OR ', $docIds),
-            $currentDate->format('Y-m-d\TH:i:s\Z')
-        );
-
-        $query = $this->client->createSelect();
-        $query->setQuery($queryPhrase);
-        $result = $this->client->select($query);
+        $urisAll = [];
+
+        $docIdsParts = array_chunk($docIds, 100);
```

Comment:
I really don't like harcoding values like this. Can you move 100 to variable or class const? Let's name it like queryLimit or maxSolrQueryParams. That will say us why it's there.